snapshot/bundlestore/server.go

```diff
@@ -2,22 +2,28 @@ package bundlestore
 
 import (
 	"fmt"
-	log "github.com/Sirupsen/logrus"
 	"io"
 	"net/http"
 	"regexp"
 	"strings"
+	"time"
+
+	log "github.com/Sirupsen/logrus"
 
 	"github.com/scootdev/scoot/common/stats"
 )
 
 type Server struct {
 	store Store
+	ttl   *TTLConfig
 	stat  stats.StatsReceiver
 }
 
-func MakeServer(s Store, stat stats.StatsReceiver) *Server {
-	return &Server{s, stat.Scope("bundlestoreServer")}
+// Make a new server that delegates to an underlying store.
+// TTL may be nil, in which case defaults are applied downstream.
+// TTL duration may be overriden by request headers, but we always pass this ttl key to the store.
```

Comment:
I tripped over 'but we always pass this ttl key to the store'... interpreted 'this' as ttl in argument - which implies that the server's ttl overrides ttl in request header....
remove  'but we always.... ' from comment